packages/terra-functional-testing/lib/webpack-server/webpack-server.js

```diff
@@ -0,0 +1,139 @@
+const WebpackDevServer = require('webpack-dev-server');
```

Comment:
could we name this file index.js ?

Reply:
Added an index.js file that exports the service, but I kept the original webpack-server.js file. This is a similar pattern to what is being done in terra-application and I think it makes sense. This allows the consistent shortcut import without dumping all of the code into the index.js file

https://github.com/cerner/terra-toolkit/pull/417/commits/7b43d2d93a6c308d7990995e74934d87b098aace